sysfs/net_class.go

```diff
@@ -113,71 +137,140 @@ func (nc NetClass) parseNetClassIface(devicePath string) (*NetClassIface, error)
 	}
 
 	for _, f := range files {
-		if !f.Mode().IsRegular() {
```

Comment:
Maybe check here for the statistics dir instead?
```golang
if !f.Mode().IsDir() && f.Name() == "statistics" {
    readstatsFiles()
    continue
}
```

Reply:
I'll make the changes, thanks for the feedback, perhaps a bit rushed to get this to work for a project.